manifests/server/config.pp

```diff
@@ -106,9 +106,20 @@
     }
   }
 
+  if $facts['selinux'] {
+    selinux::port { 'postgresql_custom_port':
```

Comment:
it would be nice if this could be enabled/disabled with a new parameter like `$postgresql::manage_selinux`. Many people I know already manage selinux settings in their profiles.

Reply:
@bastelfreak I can add that to the manifest.  Should the default value be set to false?

Reply:
My personal opinion would be that the default should be false, otherwise a major release would be required. But that's something that the module owners should decide.